releases/changelogs/changelog from 2.7.0-RC-2 to 2.7.0-RC-3.md

```diff
@@ -66,4 +66,4 @@
 >
 
 # Credits
-A special thanks to @Ableytner, @wlhlm, who contributed to this release!
\ No newline at end of file
+Special thanks to @Ableytner, @Alexdoru, @Cleptomania, @Connor-Colenso, @FourIsTheNumber, @Glease, @ah-OOG-ah, @kurrycat2004, @lordIcocain, @serenibyss, @slprime, @wlhlm, who contributed to this release!
```

Comment:
is this supposed to be everyone between all repos? because it isn't.

Reply:
It's everyone mentioned earlier in the changelog. I don't see anyone missing. The config repo does not show up in the changelog, but that's not related to this PR.

Reply:
that is very misleading then

Reply:
and it is related to this PR.
because this PR changes things from _not giving credit_ to _stealing credit_. The first is fairly minor in my book, the second absolutely not ok.

Reply:
example of credit being stolen?

Reply:
then it's not credit stealing, it's missing credits. We don't steal credits here.

Reply:
previously it was by mod, so it was missing as you say. But now it pretends to be complete so its stealing.

and yea us not doing that is why this should be changed.

Reply:
unless you see missing credits for anyone who contributed to repos (except modpack repo) there is nothing wrong in this PR, as the modpack repo seems to be handled separately and is outside the scope of this PR

Reply:
So can you stop being needlessly annoying for something like this? if you are so pissed by this, you should patch it in a separate PR.

Reply:
> because this PR changes things from not giving credit to stealing credit. 

I'm assuming you mean the PR title? I can give you that it's not accurate. At the time I was only thinking in terms of the implementation, which only considers the changelog text itself.

However, I did not change the meaning of the credit message in the changelog itself, nor did I modify individual mod's credits.